packages/create-vite-app/template/src/main.tsx

```diff
@@ -0,0 +1,34 @@
+import { isValidAutomergeUrl, Repo } from '@automerge/automerge-repo'
+import { BrowserWebSocketClientAdapter } from '@automerge/automerge-repo-network-websocket'
+import { IndexedDBStorageAdapter } from "@automerge/automerge-repo-storage-indexeddb"
+import { RepoContext } from '@automerge/automerge-repo-react-hooks'
+import {next as A} from "@automerge/automerge" //why `next`? See the the "next" section of the conceptual overview
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import App from './App.tsx'
+import './index.css'
+
+const repo = new Repo({
+  network: [new BrowserWebSocketClientAdapter("ws://sync.automerge.org")],
+  storage: new IndexedDBStorageAdapter(),
+})
+
+const rootDocUrl = `${document.location.hash.substr(1)}`
```

Comment:
substr is deprecated in favour of substring (i just merged a PR that fixes this upstream)